R/Private.R

```diff
@@ -195,13 +195,17 @@ makeDataExportable <- function(x,
     )
   }
   
-  if (nrow(x) >  nrow(x %>% dplyr::select(dplyr::all_of(primaryKeyInDataModel)) %>% dplyr::distinct())) {
-    stop(
-      " - duplicates found in primary key for table ",
-      tableName,
-      ". The primary keys are: ",
-      paste0(primaryKeyInDataModel, collapse = ", ")
-    )
+  # check to see if there are primary key collision in tables that have this unique constraint
+  if (length(primaryKeyInDataModel) > 0) {
```

Comment:
@gowthamrao This change just lets us have tables specified in the the DDL without a primary key. I don't see any other way to support the VA's vocabulary table which seems to have multiple CDM version in one somehow.

Reply:
can we just do a max

Reply:
Max won't tell us what versions they are potentially using in a cdm